src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -216,6 +234,19 @@ export class BaseEditCourseForm extends React.Component {
 
     const disabled = courseInReview || !editable;
 
+    const cancelButton = (
+        <button
+          onClick={() => {
+            this.setCollapsible(false);
+            this.props.reset();
+            this.resetOpenCourseRuns();
+          }}
+          className="btn btn-outline-primary"
+        >
+          Cancel
```

Comment:
I think `Cancel` on its own is misleading in this context. We are now clearing data instead of leaving the form. I would argue for something like `Cancel Edits` or `Clear Edits`